docs/source/concepts/AbsorptionAndMultipleScattering.rst

```diff
@@ -697,7 +697,10 @@ Multiple Scattering Correction Algorithms in Mantid Table
 | :ref:`CalculateCarpenterSampleCorrection <algm-CalculateCarpenterSampleCorrection>` | E           | NI         | Cylinder                        | Wavelength           | IA,EA,FI            ||  Only applicable to Vanadium                                                         |
 |                                                                                     |             |            |                                 |                      |                     ||  In-plane only                                                                       |
 +-------------------------------------------------------------------------------------+-------------+------------+---------------------------------+----------------------+---------------------+---------------------------------------------------------------------------------------+
-| :ref:`CalculateMultipleScattering <algm-CalculateMultipleScattering>`               | E           | MC         | Any Shape                       | Wavelength           | EA, FI, PI          ||  Based on Monte Carlo program DISCUS [16]_ (or MINUS). [17]_                         |
+| :ref:`CarpenterSampleCorrection <algm-CarpenterSampleCorrection>`                   | E           | NI         | Cylinder                        | Wavelength           | IA,EA,FI,W          ||  Only applicable to Vanadium                                                         |
```

Comment:
Carpenter is actually an approximation using spherical Bessel functions. It should be labeled as analytical. This is similar to Mayers which uses a Chebyshev polynomial to approximate things.

Reply:
OK. I didn't change this in the PR but I can change the NI to A while I'm doing a few changes in this file